Vagrantfile_RPM_Test

```diff
@@ -0,0 +1,90 @@
+# -*- mode: ruby -*-
+# # vi: set ft=ruby :
+
+# Require Vagrant 2.0+.
+Vagrant.require_version '>= 2.0.0'
+
+# Setting up globals from YAML configuration file.
+settings = YAML::load_file('config.yml')
+$pg_yum_repo = settings.fetch('yum-repos')['hoot-pgdb95-repo']
+$hoot_yum_repo = settings.fetch('yum-repos')['hoot-develop-repo']
+
+Vagrant.configure("2") do |config|
+    tomcatPort = ENV['TOMCAT_PORT']
+    if tomcatPort.nil?
+        tomcatPort = '8888'
+    end
+
+    transPort = ENV['NODEJS_PORT']
+    if transPort.nil?
+        transPort = '8094'
+    end
+
+    mergePort = ENV['P2P_PORT']
+    if mergePort.nil?
+        mergePort = '8096'
+    end
+
+    disableForwarding = ENV['DISABLE_VAGRANT_FORWARDING']
+    if disableForwarding.nil?
+        config.vm.network "forwarded_port", guest: 8080, host: tomcatPort
+        config.vm.network "forwarded_port", guest: 8094, host: transPort
+        config.vm.network "forwarded_port", guest: 8096, host: mergePort
+    end
+    
+    config.vm.provider "virtualbox" do |vb|
+        vb.gui = false
+        vb.memory = 10240
+        vb.cpus = 4
+    end
+    
+    def aws_provider(config, os)
+        config.vm.provider :aws do |aws, override|
+            override.nfs.functional = false
+            aws.instance_type = ENV.fetch('AWS_INSTANCE_TYPE', 't2.xlarge')
+            aws.block_device_mapping = [{ 'DeviceName' => '/dev/sda1', 
+                                               'Ebs.VolumeSize' => 12,
+                                               'Ebs.DeleteOnTermination' => true }] 
+            if ENV.key?('AWS_KEYPAIR_NAME')
+                aws.keypair_name = ENV['AWS_KEYPAIR_NAME']
+            end
+ 
+            if ENV.key?('AWS_SECURITY_GROUP')
+                aws.security_groups = ENV['AWS_SECURITY_GROUP']
+            end
+
+            aws.tags = {
+                'Name' => ENV.fetch('AWS_INSTANCE_NAME_TAG', 
+                                    "jenkins-hootenanny-rpms-#{os.downcase}"),
+
+                'URL'  => ENV.fetch('AWS_INSTANCE_URL_TAG', 
+                                    "https://github.com/ngageoint/hootenanny-rpms")
+            }
+
+            if ENV.key?('JOB_NAME')
+                aws.tags['JobName'] = ENV['JOB_NAME']
+            end
+
+            if ENV.key?('BUILD_NUMBER')
+                aws.tags['BuildNumber'] = ENV['BUILD_NUMBER']
+            end
+        end
+    end
+
+    # set VM operating system to CentOS7
+    config.vm.define "default", primary: true do |hoot_centos7|
+        config.vm.box = "hoot/centos7-hoot"
```

Comment:
I think it might be a better test with `hoot/centos7-minimal`.  The one you have is preprovisioned for compiling hoot.  The minimal is more representative of the system state when someone would install from RPM.

Reply:
Ok, I can substitute the box snapshot